### PR TITLE
sdk/js: Add deprecated warning to README

### PR DESCRIPTION
**Pull request by kev1n-peters**

### CODE REVIEW

**Review by evan-gray — DISMISSED** [review]

**Review by panoel — DISMISSED** [review]

**Review by artursapek — DISMISSED** [review]

**Comment by kev1n-peters** [comment]
Waiting to deprecate until this is merged: https://github.com/wormhole-foundation/wormhole-sdk-ts/pull/602

File: sdk/js/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# Wormhole SDK
+# Wormhole SDK (Deprecated)
 
-> Note: This is a pre-alpha release and in active development. Function names and signatures are subject to change.
+**⚠️ This SDK is deprecated. Please use the new SDK available at [wormhole-sdk-ts](https://github.com/wormhole-foundation/wormhole-sdk-ts).**
```
Comment (tonyjin) [inline sdk/js/README.md:3]: Recommend linking to NPM instead as most users would be looking at this via npmjs.org:

```Please use the new SDK available at [@wormhole-foundation/sdk](https://www.npmjs.com/package/@wormhole-foundation/sdk).```

**Review by evan-gray — APPROVED** [review]
reminder to also deprecate this on npm
https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions#deprecating-an-entire-package

**Comment by tonyjin** [comment]
Closed in favor of #4142

**Pull request closed without merging**